src/main/java/Deadline.java

```diff
@@ -0,0 +1,28 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class representation of Deadline
```

Comment:
The first line of every Javadoc should end with a period '.' I made that same mistake too but my peer reviewer pointed it out to me!